drupal/rootfs/etc/confd/templates/setup-environment.sh.tmpl

```diff
@@ -33,7 +33,7 @@ multisubstitute
   define DB_USER "drupal_default"
   define EMAIL "webmaster@localhost.com"
   define INSTALL "true"
-  define INSTALL_EXISTING_CONFIG "false"
+  define INSTALL_EXISTING_CONFIG "true"
```

Comment:
Maybe override it in the demo image as there is no configuration in the base Drupal image.